### DISCUSSION TITLE
Encode Club x Akash Club Education Series

### DISCUSSION THREAD

**Discussion by anthonyencodeclub** [akash-network/community#458] [Governance Proposals]
Hi Everyone!

We've been big fans of Akash from the outside now and wanted to jump in to get feedback on running an Education initiative to drive developers to Akash!

Encode Club, the community I run, has been fostering developer education in web3 for over 5 years now! Our main focus to engage and onboard developers is around high-quality live content and support. We work with most of the leading protocols as long-term clients! We now train over 20k developers per quarter and have had over 100 vc-funded startups out of Encode such as Euler Finance!

We'd like to run a public education series of 8 events across 8 weeks to bring 2,000 developers to Akash, ready to build in the hackathons and other events the ecosystem is supporting. Encode will be responsible for all marketing, programme management, curriculum and speaker arrangement. We'd also love to engage as much of the community as possible in this effort.

We envisage starting this initiative in April, allowing plenty of time for the community to digest, engage and market this programme! We'd like to call it 'Akash Club'.

Our track record of this sort of event includes:

[Sui Educate](https://www.encode.club/sui-educate)
[Polkadot Educate](https://www.encode.club/polkadot-educate)
[Internet Computer Educate](https://www.encode.club/zero-to-dapp-educate)
[Near Educate](https://www.encode.club/near2023)

A deck is [attached.](https://drive.google.com/file/d/19UQh1BjgEtf1JZwF1-9HgMJYjzlSsUB2/view?usp=sharing)

For running this initiative, we would request a grant of $25,000 to help support the setup of this programme!

We would love to discuss with the community!

**Comment by CaptainSplince** [comment]
Hey Anthony - Thanks for the proposal! I have a few questions stemming from your overview.

- Your website says you're community has 100,000 developers in it, but the Akash deck you linked says you have 500,000. Just a little confused on the **total size of your community**.

- How much of this program would be done in-house and how much would you require Akash network community members or Overclock employees to participate? **Is this basically, pay the fee and you handle the rest?**

- I think it's important to include a detailed framework for how the $25k would be spent and what the estimated outcomes will be aligning with historical outcomes from previous initiatives with other similar projects. eg - **how many new community members can we expect to join our community?** not only join but **begin activly participating.**

- You mention that you want to host 8 events across 8 weeks to bring 2,000 developers to Akash. What are these events and what does "bring 2000 developers to Akash" mean? Historically speaking, do those 2000 developers start activly building on the various protocols you have partnered with? I'd love to better understand the end results of these initiatives. **What's the funnel look like?**

- Looks like the focus of your programs are around "high-quality live content and support." What does support look like? **How do you support developers outside of live events?** 

- In the Akash Deck, you mention 5 seperate programs. Will all of these programs be part of this proposal? **A detailed timeline would be useful for review.**

- From what I see this is really an "awareness"  campaign. I don't see any long term thinking in the proposal eg - **What happens after the 8 weeks is over?** Do the developers who participate have a support system in place?

Thanks Anthony

**Comment by anthonyencodeclub** [comment]
Hey!

First, thank you for these considered questions!

- Website needs updating! It's 500,000! We are scaling a lot atm!
- We always handle everything!
- This programme should engage about 2000 developers. How they actively
participate depends on the opportunities within that ecosystem but if there
were, say, a hackathon you could expect 25-50% to convert in the short-term
to building projects! We've seen this across many ecosystems including
Solana, Polkadot, Near and many more!
- See answer above!
- We supposed develops in several ways: 1) during the programme, we have
dedicated programme managers to answer administrative questions and help
them have a great experience 2) we have a dedicated talent and investment
team for jobs or/and raising money 3) we have an in-house technical team to
answer technical questions!
- That was just an example of what else we do, this programme would place
take over 8 weeks, 1 session per week with 6 weeks marketing beforehand.
Subject to being funded, we'd aim to start the marketing process by the end
of Q1 with the educations starting in Q2!
- This is the first step of a broader initiative. If successful, we would
follow it up with a hackathon and then accelerator (in a separate funding
request). Our focus is actually always towards long-term building and
engagement (as evidenced by our long-term partnerships with many
blockchains) but we know we need to build trust with new ecosystems before
scaling!

I'm happy to answer any others here or on call, and adjust our proposal
accordingly!

Best,
Anthony



Anthony Beaumont

CEO and co-founder

+44 7590 213 269

***@***.***
<https://www.linkedin.com/in/anthony-beaumont-623894164/>
<https://twitter.com/anth0nybeaumont?lang=en>

www.encode.club

**Comment by brewsterdrinkwater** [comment]
this discussion is inactive.
